src/_data/site.json

```diff
@@ -10,7 +10,7 @@
     },
     "messaging": {
         "tagLine": "Unlock Industrial Data. Integrate Everything. Optimize Faster.",
-        "subtitle": "FlowFuse is the Industrial Application Platform empowering teams to build, deploy and manage applications that optimize industrial operations.",
+        "subtitle": "FlowFuse is the industrial application platform that brings the exclusive AI-enhanced Node-RED to the market, empowering teams to build, deploy, and manage solutions that optimize operations.",
```

Comment:
Why no more capitals on Industrial Application Platform?

Reply:
Not sure about the NR mention here, if it's about the outcome first and foremost and about the technology second.